.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
For consistency, readability, and following pep 8, I am fine with this. I understand your argument @bogdanm, everyone has a different preferences.

Anyone else against this, speak now! :-)

Note: I am more flexible with this rule my codebase, I tend to follow 80 but not a requirement (once in a while to have line >80 is fine).